.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,65 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Create a temporary Git tag"
+        run: git tag "$MARKETING_VERSION"
+
+      - name: "Extract version information from project"
+        run: |
+          MARKETING_VERSION=$(xcrun xcodebuild -showBuildSettings -project Sparkle.xcodeproj | grep MARKETING_VERSION | sed -E 's/.+=[[:space:]]+(.*)$/\1/')
```

Comment:
I think multiple results are going to be matched now because of `MARKETING_VERSION=\"$(MARKETING_VERSION)\"` in the exported preprocessor definitions, but please re-check and handle this as necessary.

Reply:
Also possibly the comment about being careful to use `MARKETING_VERSION`..

Reply:
I think in your `grep` part you need to make sure the line starts with `MARKETING_VERSION`. At least that would be one way of doing it.

Reply:
Eg

`grep -m 1 '^[[:space:]]*CURRENT_PROJECT_VERSION'`

Reply:
Whatever improvement done here should likely also apply to extracting `SPARKLE_VERSION_SUFFIX`

Reply:
Your grep command seems to work where my sed doesn't, so I've updated to use that: 2daa27b58e88a6d31235fa46bf7e270706f7711f